tests/bluetooth/controller/ctrl_periodic_sync/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# some of the control procedures in the BT LL depend on
+# the following configs been set
+
+config SOC_COMPATIBLE_NRF
+	default y
+
+config BT_CTLR_DATA_LEN_UPDATE_SUPPORT
+	default y
+
+config BT_CTLR_PHY_UPDATE_SUPPORT
+	default y
+
+config BT_CTLR_PHY_CODED_SUPPORT
+	default y
+
+config BT_CTLR_PHY_2M_SUPPORT
+	default y
+
+config ENTROPY_NRF_FORCE_ALT
+	default n
+
+config ENTROPY_NRF5_RNG
+	default n
```

Comment:
What is preventing us from enabling these in `prj.conf`?

Reply:
None, however this is the approach that is done for all the controller tests.

Reply:
But I have to assume there were reasons why those couldn't be enabled in the other tests, otherwise this is also just a bug waiting to happen since we aren't respecting dependencies which can affect behavior of the functions we are testing. 

@cvinayak what's your though here?

Reply:
1. `_SUPPORT` ones are hidden and can only be overridden using an application Kconfig file.
2. the none hidden ones can be overridden in prj.conf, but these are again being `select` and hence had to be overridden by a Kconfig file, i am guessing.